utils/config/daos_server.yml

```diff
@@ -368,3 +376,11 @@
 #  # block devices that should be different across different server instance.
 #  bdev_class: kdev
 #  bdev_list: [/dev/sdc,/dev/sdd]
+#
+#  # If Volume Management Devices (VMD) are to be used, then the use_vmd flag
+#  # needs to be set. The bdev_class will remain the default "nvme" type,
+#  # and similar to NVMe, bdev_list will include the NVMe SSD address behind
+#  # the VMD. vmd_include also needs to be set to the VMD address in order for
```

Comment:
where do they set vmd_include?

Reply:
I think vmd_include has been removed (was part of a previous iteration of the patch I did). Although it brings up a good point about whether or not we should include this variable (similar to bdev_include) giving the user the option to not unbind all VMD addresses available on the system if they don't want to for some reason. A good example is the issue I am having on wolf-156 now where unbinding 0000:85:05.5 is causing issues and crashing the node

Reply:
they don't, it's been removed